app/src/main/java/org/wikipedia/page/PageActivity.java

```diff
@@ -686,8 +687,9 @@ public void onActivityResult(int requestCode, int resultCode, final Intent data)
         } else if (requestCode == Constants.ACTIVITY_REQUEST_IMAGE_CAPTION_EDIT
                 && resultCode == RESULT_OK) {
             pageFragment.refreshPage();
-            FeedbackUtil.showMessage(this, TextUtils.isEmpty(pageFragment.getLeadImageEditLang()) ? getString(R.string.description_edit_success_saved_image_caption_snackbar)
-                    : getString(R.string.description_edit_success_saved_image_caption_in_lang_snackbar, pageFragment.getLeadImageEditLang()));
+            FeedbackUtil.makeSnackbar(this, TextUtils.isEmpty(pageFragment.getLeadImageEditLang()) ? getString(R.string.description_edit_success_saved_image_caption_snackbar)
+                    : getString(R.string.description_edit_success_saved_image_caption_in_lang_snackbar, app.language().getAppLanguageLocalizedName(StringUtils.defaultString(pageFragment.getLeadImageEditLang(), app.language().getAppLanguageCodes().get(0)))), FeedbackUtil.LENGTH_DEFAULT)
```

Comment:
Is it possible to use `app.language().getAppLanguageCode()` instead of `app.language().getAppLanguageCodes().get(0)`?